features/step_definitions/functional_tests/functional_tests_steps.rb

```diff
@@ -0,0 +1,58 @@
+Given(/^the test package "([^"]*)" is installed on the controller node$/) do |test_package|
+  control_node.rpm_q(test_package)
+end
+
+Given(/^the package "([^"]*)" is installed on the controller node$/) do |client_package|
+  control_node.rpm_q(client_package)
+end
+
+Given(/^the proper cirros test image has been created$/) do
+  test_image_name = "cirros-test-image-uec"
```

Comment:
doesn't barclamp-tempest upload it under that name? "Given the proper cirros test image has been created" doesn't sound to me as if cct would be uploading the image, but rather that it checks for the image to exist..